### ISSUE TITLE
mytharchive terminates when applying a menu to recording

### ISSUE THREAD

**Issue by bshanteau** [MythTV/mythtv#626]
* **Platform**:
`Linux dvr 5.15.0-46-generic #49-Ubuntu SMP Thu Aug 4 18:03:25 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux`

* **MythTV version**:
`2:33.0~master.202208171747.a6e1128e0f~ubuntu22.04.1`

* **Package version**:
PPA: mythbuntu/33

* **Component**:
mythburn.py
mytharchivehelper

### What steps will reproduce the bug?
MythFrontend->Utilities->Archive Files->Create DVD ->Select Destination: File: Make ISO Image->DVD Menu Theme: MythCenter - Animated->Select Archive Items: Add Recording-->Select recording->Next->Menu->Next

### How often does it reproduce? Is there a required condition?
Every time.

### What is the expected behaviour?
Create iso file with menu
[mythburn.log](https://github.com/MythTV/mythtv/files/9386577/mythburn.log)
[work.zip](https://github.com/MythTV/mythtv/files/9386595/work.zip)

### What do you see instead?
Terminates without creating iso file.

### Additional information
Last few lines of mythburn.log:

mytharchivehelper -q -q --createthumbnail --infile "/storage/mytharchive/work/1/stream.mv2" --thumblist '0,0,0,0,0,0,0,0' --outfile "/storage/mytharchive/work/1/chapter-%1.jpg"
Segmentation fault (core dumped)
`************************************************************`
ERROR: Failed while running mytharchivehelper to get thumbnails.
Result: 139, Command was mytharchivehelper -q -q --createthumbnail --infile "/storage/mytharchive/work/1/stream.mv2" --thumblist '0,0,0,0,0,0,0,0' --outfile "/storage/mytharchive/work/1/chapter-%1.jpg"
See mythburn.log for more information.
`************************************************************`
Terminated

**Comment by rcrdnalor** [comment]
Since you run mythtv master on Ubuntu, please check
if the `mythtv-dbg` package is installed/installable.
See
https://launchpad.net/~mythbuntu/+archive/ubuntu/33
https://www.mythtv.org/wiki/Debugging
for further information.

The ticket how-to suggests that you should  post a
backtrace of the segfault you got on your system:
See
https://www.mythtv.org/wiki/Trac:TicketHowTo
Please keep the source, the reason of your segfault 
for later posting on request of the MythTV developer.

In case you need further information, please search
the mailingt list
https://lists.archive.carbon60.com/mythtv/
or the forum
https://forum.mythtv.org/
for getting a proper backtrace.

In this case, please post the backtrace of
```
mytharchivehelper -q -q --createthumbnail --infile "/storage/mytharchive/work/1/stream.mv2" --thumblist '0,0,0,0,0,0,0,0' --outfile "/storage/mytharchive/work/1/chapter-%1.jpg"
```
when runnning under gdb.

**Comment by bshanteau** [comment]
> Since you run mythtv master on Ubuntu, please check
> if the `mythtv-dbg` package is installed/installable.

@rcrdnalor I cannot install `mythtv-dbg`. In Synaptic I get:

```
mythtv-dbg:

Package mythtv-dbg has no available version, but exists in the database.
This typically means that the package was mentioned in a dependency and never uploaded, has been obsoleted or is not available with the contents of sources.list```

**Comment by rcrdnalor** [comment]
Ask the forum or the mailing lists.

**Comment by kmdewaal** [comment]
@bshanteau  can you please check if this problem still exists after the fix of #633 has been verified?

**Comment by rcrdnalor** [comment]
If you do not answer the above question, this issue will be closed, soon.

**Comment by bshanteau** [comment]
@kmdewaal wrote:
> @bshanteau can you please check if this problem still exists after the fix of #633 has been verified?

It's working fine now. Here is the log for the successful making of an iso with a menu from 7 recordings despite hundreds of warning messages for some of the recordings:
[mythburn.log.zip](https://github.com/MythTV/mythtv/files/10101469/mythburn.log.zip)

**Comment by kmdewaal** [comment]
OK, closing this ticket as the problem is solved.
